### PR TITLE
Remove adding time to adtech date range causing date confusion

### PR DESCRIPTION
**Pull request by minsikzzang**
Remove adding time to adtech date range causing date confusion

### CODE REVIEW

File: spec/report_spec.rb [hunk 1]
```diff
@@ -147,16 +147,9 @@
...
 
   describe '.gregorian_calendar' do
-    context 'when given time is end of day' do
-      it 'should return relevant gregorian_calendar object' do
-        cal = @report.send(:gregorian_calendar, 2016, 5, 12, 23, 59, 59)
-        expect(cal.getTime.to_s). to eq('Sun Jun 12 23:59:59 BST 2016')
-      end
-    end
-
     context 'when given time is beginnig of day' do
```
Comment (willhall88) [inline spec/report_spec.rb]: spelling `beginning`

File: spec/report_spec.rb [hunk 2]
```diff
@@ -147,16 +147,9 @@
   end
 
   describe '.gregorian_calendar' do
-    context 'when given time is end of day' do
```
Comment (willhall88) [inline spec/report_spec.rb]: why is this test being removed?

Comment (willhall88) [inline spec/report_spec.rb]: ah sorry. I get it now. There is no time anymore!! so only day!

**Comment by willhall88** [comment]
👍

File: spec/report_spec.rb (hunk 2 above)
Comment (minsikzzang) [inline spec/report_spec.rb]: hmm, we used to care about time (hour / min / sec) for date range for adtech api but that caused date range confusion / timezone issue, so hard coded to `00:00:00` for all date ranges, it seems adtech api don't care time for date ranges.

**Comment by ericescalante** [comment]
LGTM

**Pull request merged**